internal/build/docker_test.go

```diff
@@ -74,9 +58,12 @@ func TestMount(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	// TODO(dmiller): this is slow
-	f.assertFileInImageWithContents(tag, "/src/hi/hello", "hi hello")
-	f.assertFileInImageWithContents(tag, "/src/sup", "my name is dan")
+	pcs := []pathContent{
```

Comment:
hmm it feeeeels like we should a single struct that holds test data that we can both write files from and assert test behavior from...? any thoughts? seems awk to rewrite the same string multiple times :-/

Reply:
yeah this would be better if we were doing table tests...